packages/devextreme/testing/tests/DevExpress.ui.widgets/chat.tests.js

```diff
@@ -347,6 +350,123 @@ QUnit.module('renderMessage', moduleConfig, () => {
     });
 });
 
+QUnit.module('onMessageSend', moduleConfig, () => {
+    QUnit.test('New message should be created after clicking the send button if there is text', function(assert) {
```

Comment:
That's right, because I'm not realizing that scenario right now. We should use registerKeyHandler or something similar in the TextBox to catch Enter keystrokes and run the method that sends the messages. This is not implemented, so it's not tested

Reply:
Well, this task should be added in your // todo list